backend/app/api/endpoints/admin/public_teams.py

```
@@ -4,25 +4,162 @@
+    ):
+        return None
+    return asset
+
+
+@router.post(
+    "/public-teams/icon-assets",
+    response_model=PublicTeamIconUploadResponse,
+    status_code=status.HTTP_201_CREATED,
+)
+async def upload_public_team_icon(
+    file: UploadFile = File(...),
```

Ruff (0.16.0)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)